astropy/io/misc/asdf/conftest.py

```diff
@@ -5,7 +5,7 @@
 
 def get_asdf_tests():
     asdf_dir = Path(__file__).parent.resolve()
-    paths = Path(asdf_dir).rglob("test_*.py")
+    paths = Path(asdf_dir).rglob("*.py")
```

Comment:
This change makes `get_asdf_tests` function name misleading. Please add a comment to explain.

Reply:
Thanks for taking a look. I added a comment. Let me know if that's what you were looking for.